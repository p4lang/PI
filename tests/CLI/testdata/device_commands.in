show_devices
remove_device 8589934592
show_devices
