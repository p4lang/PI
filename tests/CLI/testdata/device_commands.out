????
Device assigned successfully.
Selecting device.
????
Showing devices:
8589934592 (selected)
????
Device removed successfully.
????
Showing devices:
????
